app/src/features/projects/view/ProjectDetailsPage.tsx

```diff
@@ -72,20 +87,26 @@ const ProjectDetailsPage: React.FC<IProjectDetailsProps> = (props) => {
   const classes = useStyles();
 
   return (
-    <Box className={classes.projectMetadata}>
-      <Box component="section" p={3}>
+    <Box className={classes.projectMetadata} p={3}>
+
+      <Box mb={3}>
+        <Typography variant="h2">Project Details</Typography>
+      </Box>
+
+      <Box component="section">
         <Typography variant="body1" component={'h3'} data-testid="GeneralInfoTitle">
           General Information
         </Typography>
+        <Divider></Divider>
         <GeneralInformation projectForViewData={projectForViewData} codes={codes} refresh={refresh} />
       </Box>
 
-      <Divider></Divider>
 
-      <Box component="section" p={3}>
+      <Box component="section">
         <Typography variant="body1" component={'h3'} data-testid="ContactsTitle">
           Project Contacts
         </Typography>
+        <Divider></Divider>
```

Comment:
Is there a reason we don't write `<Divider />`? I'm guessing for consistency?

Reply:
Removed as this component isn't used anymore